[2024-11-29 16:35:15,808] [INFO] [real_accelerator.py:219:get_accelerator] Setting ds_accelerator to cuda (auto detect)
[2024-11-29 16:35:17,058] [WARNING] [runner.py:215:fetch_hostfile] Unable to find hostfile, will proceed with training with local resources only.
[2024-11-29 16:35:17,059] [INFO] [runner.py:607:main] cmd = /home/nsatch/cs598aie/aie_spec_dec/ashley_new_code/decoding-speculative-decoding/venvs/spec_dec/bin/python3.9 -u -m deepspeed.launcher.launch --world_info=eyJsb2NhbGhvc3QiOiBbMF19 --master_addr=127.0.0.1 --master_port=29500 --enable_each_rank_log=None spec_decoding_final_param.py
[2024-11-29 16:35:18,125] [INFO] [real_accelerator.py:219:get_accelerator] Setting ds_accelerator to cuda (auto detect)
[2024-11-29 16:35:19,297] [INFO] [launch.py:146:main] WORLD INFO DICT: {'localhost': [0]}
[2024-11-29 16:35:19,297] [INFO] [launch.py:152:main] nnodes=1, num_local_procs=1, node_rank=0
[2024-11-29 16:35:19,297] [INFO] [launch.py:163:main] global_rank_mapping=defaultdict(<class 'list'>, {'localhost': [0]})
[2024-11-29 16:35:19,297] [INFO] [launch.py:164:main] dist_world_size=1
[2024-11-29 16:35:19,297] [INFO] [launch.py:168:main] Setting CUDA_VISIBLE_DEVICES=0
[2024-11-29 16:35:19,303] [INFO] [launch.py:256:main] process 445864 spawned with command: ['/home/nsatch/cs598aie/aie_spec_dec/ashley_new_code/decoding-speculative-decoding/venvs/spec_dec/bin/python3.9', '-u', 'spec_decoding_final_param.py', '--local_rank=0']
[2024-11-29 16:35:20,402] [INFO] [real_accelerator.py:219:get_accelerator] Setting ds_accelerator to cuda (auto detect)
1 0 0
[2024-11-29 16:35:22,306] [INFO] [launch.py:319:sigkill_handler] Killing subprocess 445864
[2024-11-29 16:35:22,307] [ERROR] [launch.py:325:sigkill_handler] ['/home/nsatch/cs598aie/aie_spec_dec/ashley_new_code/decoding-speculative-decoding/venvs/spec_dec/bin/python3.9', '-u', 'spec_decoding_final_param.py', '--local_rank=0'] exits with return code = 1
Return status: 0


The runtime is 8.299890756607056